pkg/development/assertions.go

```diff
@@ -42,14 +43,18 @@ func runAssertions(devContext *DevContext, assertions []blocks.Assertion, expect
 	for _, assertion := range assertions {
 		tpl := tuple.MustFromRelationship[*v1t.ObjectReference, *v1t.SubjectReference, *v1t.ContextualizedCaveat](assertion.Relationship)
 
+		// NOTE: zeroes are fine here to mean "unknown"
```

Comment:
If safecast can't successfully cast, it returns zero. For pointing at line numbers, I think this should be fine (especially since you'd have to do something weird to end up out of bounds here).

Reply:
We should either log the error or have it panic